application/views/index.php

```diff
@@ -1 +1,27 @@
-<h1>欢迎参加北大车协全国高校山地车交流赛</h1>
+<body style="background:url('http://ww2.sinaimg.cn/large/0063uqpDjw1eqnm3enisyj31kw11xarl.jpg')top center;"><center>
```

Comment:
body tag is alreay contained in `header.php`.
Please use css to set background.